build.fsx

```diff
@@ -131,16 +132,70 @@ Target "MultiNodeTests" (fun _ ->
 
 Target "NBench" <| fun _ ->
     CleanDir outputPerfTests
-    // .NET Framework
-    let testSearchPath =
+    // .NET Framework -------------------------------------
+    let testSearchPathNet452 =
+        let assemblyFilter = getBuildParamOrDefault "spec-assembly" String.Empty
+        sprintf "src/**/bin/Release/net452/**/*%s*.Tests.Performance.dll" assemblyFilter
+
+    let nbenchTestPathNet452 = findToolInSubPath "NBench.Runner.exe" (toolsDir @@ "net452" @@ "NBench.Runner*")
+    let nbenchTestAssembliesNet452 = !! testSearchPathNet452
+    printfn "Using NBench.Runner: %s" nbenchTestPathNet452
+
+    let runNBenchNet452 assembly =
+        let spec = getBuildParam "spec"
+        let teamcityStr = (getBuildParam "teamcity")
+        let enableTeamCity = 
+            match teamcityStr with
+            | null -> false
+            | "" -> false
+            | _ -> bool.Parse teamcityStr
+
+        let args = StringBuilder()
+                |> append assembly
+                |> append (sprintf "output-directory=\"%s\"" outputPerfTests)
+                |> append (sprintf "concurrent=\"%b\"" true)
+                |> append (sprintf "trace=\"%b\"" true)
+                |> append (sprintf "teamcity=\"%b\"" enableTeamCity)
+                |> toText
+
+        let result = ExecProcess(fun info -> 
+            info.FileName <- nbenchTestPathNet452
+            info.WorkingDirectory <- (Path.GetDirectoryName (FullName nbenchTestPathNet452))
+            info.Arguments <- args) (System.TimeSpan.FromMinutes 45.0) (* Reasonably long-running task. *)
+        if result <> 0 then failwithf "NBench.Runner failed. %s %s" nbenchTestPathNet452 args
+    
+    nbenchTestAssembliesNet452 |> Seq.iter (runNBenchNet452)
+
+    // .NET Core ---------------------------------------
+    // Publish netcoreapp1.1 performance test assemblies
+    let performanceTestProjects = !! "./**/*.Tests.Performance.csproj"
```

Comment:
Each test run for performance tests take 45 minutes right now, we can't just double it. We have to run them separately